docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -10,24 +10,21 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the ri
 
 [NOTE]
 ====
-The procedure described below uses a built-in questionnaire for assessing the risks involved in containerizing an application. By default, this procedure is required prior to reviewing the application. However, you might want to skip this step and use your own questionnaire instead. In that case, use the Skipping assessment option.
+The following procedure uses a built-in questionnaire for assessing the risks involved in containerizing an application. Before reviewing the application, you should perform this procedure by default.
```

Comment:
1) Comment # 1
Replace this - 
The following procedure uses a built-in questionnaire for assessing the risks involved in containerizing an application.

With this -
A built-in questionnaire is provided for assessing the risks involved in containerizing an application. Custom questionnaires can also be used. 

2) Comment # 2
This statement needs to be removed since this doesn't hold good anymore .
Before reviewing the application, you should perform this procedure by default.

3) Is there a section on custom questionnaires ? If so, could you please point me to it ? I'm available on Slack/Google chat for discussions .